<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QMainWindow" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_18">
     <item row="0" column="0">
      <widget class="QLabel" name="emailLabel_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>email       </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emailLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="placeholderText">
        <string>email must contain @gmail.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_17">
     <item row="0" column="0">
      <widget class="QLabel" name="firstnameLabel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>first name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstnameLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="placeholderText">
        <string>firstname must contain a_z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_14">
     <item row="0" column="0">
      <widget class="QLabel" name="lastnameLabel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>last name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lastnameLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="placeholderText">
        <string>lastname must contain a_z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_15">
     <item row="0" column="0">
      <widget class="QLabel" name="emailLabel_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>phone number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="phonenumberLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="placeholderText">
        <string>example: 989131234567</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_13">
     <item row="0" column="1">
      <widget class="QLineEdit" name="passwordLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>password must contain a_z or 0_9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="passwordLabel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="signupPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>signup</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_16">
     <item row="0" column="0">
      <widget class="QLabel" name="usernameLabel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameLineEdit_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);</string>
       </property>
       <property name="placeholderText">
        <string>username must contain a_z or 0_9</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelErrorMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
